tests/jsmarionette/runner/marionette-js-runner/host/python/runner-service/runner_service/handlers/runner.py

```diff
@@ -132,16 +134,62 @@ def __init__(self, *args, **kwargs):
     def start_runner(self, target, options):
         port = options.get('port')
         profile = Profile(profile=options.get('profile'))
-
-        print('---- start runner ----');
-        print(port, profile)
+ 
+        print("---- preparing to start runner ----")
+        print("port =", port, " with profile =", profile)
         self.runner.device.setup_port_forwarding(local_port=port, remote_port=port)
 
+        print("stopping b2g on device before starting runner.")
+        self.runner.device.dm.shellCheckOutput(['stop', 'b2g'])
```

Comment:
nit: I usually like to cache a reference to the thing in a local variable for brevity (and 80 characters). Can we do `device = self.runner.device` here and then use that ref?